content-src/reducers/setSearchContent.js

```diff
@@ -0,0 +1,59 @@
+/*globals require, module */
```

Comment:
This is singular in other files (see [ESLint "Specifying Globals"](http://eslint.org/docs/user-guide/configuring#specifying-globals)), plus not 100% certain you need them, or if they're implied if you're using an ESLint `env` of [`node: true`](https://github.com/mozilla/activity-stream/blob/c4813d2de280a5a6094d3f689da1d6c3eb5eb0f6/.eslintrc#L12).